pkg/kritis/review/review.go

```diff
@@ -170,12 +171,12 @@ func (r Reviewer) addAttestations(image string, atts []metadata.PGPAttestation,
 	}
 	keys := map[string]string{}
 	for _, auth := range auths {
-		s, err := attestation.NewPgpKey("", auth.Spec.PublicKeyData)
+		key, fingerprint, err := fingerprint(auth.Spec.PublicKeyData)
 		if err != nil {
 			glog.Errorf("Error parsing key for %q: %v", auth.Name, err)
 			continue
 		}
-		keys[auth.Name] = s.Fingerprint()
+		keys[fingerprint] = key
```

Comment:
I think this is changing the existing behavior? It should be `keys[auth.Name] = fingerprint`

Reply:
Thanks sorry for that.